src/app/item-page/versions/notice/item-versions-notice.component.ts

```diff
@@ -96,27 +90,39 @@ export class ItemVersionsNoticeComponent implements OnInit {
         startWith(false),
       );
     }
+
     // Compute the destination URL from latestVersion$ with the namespace
     this.destinationUrl$ = this.latestVersion$.pipe(
       switchMap(latestVersion => latestVersion?.item || of(null)),
       map(item => {
-        const routeCommands = [this.getItemPage(item?.payload)]; // e.g., ['/items/xyz']
+        const itemId = this.extractItemId(item?.payload); // Extract only the UUID
+
+        if (!itemId) {
+          console.error('No valid UUID found in payload');
+          return this.document.location.pathname; // Fallback to the current path if extraction fails
+        }
 
-        // Use the current ActivatedRoute to make it work like [routerLink]
-        const urlTree = this.router.createUrlTree(routeCommands, { relativeTo: this.activatedRoute });
+        // Get the base URL dynamically - with the namespace. Remove the last part of the path (the item UUID).
+        const baseUrl = this.document.location.pathname.split('/').slice(0, -1).join('/');
 
-        return this.router.serializeUrl(urlTree); // Get the final URL string
+        // Construct the final URL dynamically
+        const finalUrl = `${baseUrl}/${itemId}`;
+
+        return finalUrl;
       })
     );
   }
 
   /**
-   * Get the item page url
-   * @param item The item for which the url is requested
+   * Extracts the first valid UUID from the payload
    */
-  getItemPage(item: Item): string {
-    if (hasValue(item)) {
-      return getItemPageRoute(item);
-    }
+  extractItemId(payload: any): string | null {
+    if (!payload) { return null; }
+
```

Comment:
there is no function that does this already available? Is it so special that we are the first one to use it like this?

Reply:
Good point, there is no need to extract that UUID, thanks.